homeassistant/components/nextbus/coordinator.py

```diff
@@ -26,10 +32,10 @@ def __init__(self, hass: HomeAssistant, agency: str) -> None:
             _LOGGER,
             config_entry=None,  # It is shared between multiple entries
             name=DOMAIN,
-            update_interval=timedelta(seconds=30),
+            update_interval=timedelta(seconds=UPDATE_INTERVAL_SECONDS),
         )
-        self.client = NextBusClient(agency_id=agency)
-        self._agency = agency
+        self.client: NextBusClient = NextBusClient(agency_id=agency)
+        self._agency: str = agency
```

Comment:
```suggestion
        self.client = NextBusClient(agency_id=agency)
        self._agency = agency
```
types should be inferred here, no need for explicit typing